### DISCUSSION TITLE
在境外使用rdp，速度很慢的问题

### DISCUSSION THREAD

**Discussion by RamonBrisk** [natfrp/wiki#396] [通常话题]
目前我在加拿大经常使用rdp连接到家里的电脑， 但是有注意到有时候速度特别慢（加拿大夜里，但是其白天还好），有知道是什么原因吗， 我现在猜测是由于出口带宽限制了速度，有没有朋友能给一个分析和解决方法

**Comment by fakeboboliu** [comment]
可以尝试两个 香港CN2-GIA 节点，这是我们目前能提供对于跨境线路最优化的节点。
如果仍存在问题，那可能不是跨国造成的问题。
